.travis.yml

```
@@ -116,7 +116,9 @@ jobs:
     if: repo == "ansible/ansible-lint" AND tag IS present
     python: "3.7"
     env: &deploy-env
-      TOXENV: metadata-validation
+      TOXENV: build-dists
```

We should keep separate build and validation steps
```suggestion
      TOXENV: build-dists,metadata-validation
```